src/app/components/dashboard-menu/dashboard-menu.controller.coffee

```diff
@@ -1,12 +1,25 @@
+DashboardMenuCtrl = ($scope, MnoeCurrentUser) ->
+  'ngInject'
+
+  MnoeCurrentUser.get().then(
+    (success) ->
+      self.current_user_role = MnoeCurrentUser.user.organizations[0].current_user_role
+
+    $scope.isAdminRole = ->
+      self.current_user_role == 'Super Admin' || self.current_user_role == 'Admin'
+  )
```

Comment:
I think we can now move this code in a service. And the reference the service to be available in the whole layout:

In layout controller:
```
layout.user = MnoeCurrentUser
```